module/zfs/spa.c

```diff
@@ -2769,14 +2919,16 @@ spa_load_verify_cb(spa_t *spa, zilog_t *zilog, const blkptr_t *bp,
 	 */
 	if (!spa_load_verify_metadata)
 		return (0);
+	if (zilog && zil_shared_log(zilog))
+		io_spa = spa_get_shared_log_pool(spa);
```

Comment:
Cosmetics, but wouldn't `log_spa` or `zil_spa` be more specific?

Reply:
The idea is that this variable is the one that we will be issuing the IO to; normally, this is just the `spa` that was passed in to the function. But if this is a client pool, and this is specifically a zil block we're checking, we want to issue the IO/do the blkptr_verify in this other spa. Calling it `zil_spa` would be confusing since we're not always verifying a block in the zil.